### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ribafish**
Added sbt-ccud repo link

### CODE REVIEW

**Review by jprinet — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -47,6 +48,7 @@ The Develocity build validation scripts are open-source software released under
 [develocity-oss-projects]: https://github.com/gradle/develocity-oss-projects
 [ccud-gradle-plugin]: https://github.com/gradle/common-custom-user-data-gradle-plugin
 [ccud-maven-extension]: https://github.com/gradle/common-custom-user-data-maven-extension
+[ccud-sbt-plugin]: https://github.com/gradle/common-custom-user-data-sbt-plugin-plugin
```
Comment (erichaagdev) [inline README.md:51]:
```suggestion
[ccud-sbt-plugin]: https://github.com/gradle/common-custom-user-data-sbt-plugin
```

Comment (erichaagdev) [reply README.md:51]: All the other repos too.

Comment (ribafish) [reply README.md]: 🤦  I'll fix those

**Pull request merged**